buildpack.md

```diff
@@ -1059,9 +1053,13 @@ id = "<buildpack ID>"
 version = "<buildpack version>"
 optional = false
 
-[[stacks]]
-id = "<stack ID>"
-mixins = ["<mixin name>"]
+[[targets]]
+os = "<OS name>"
+arch = "<architecture>"
+variant = "<architecture variant>"
+[[targets.distributions]]
+name = "<OS distribution name>"
+versions = ["<OS distribution version>"]
```

Comment:
were we talking about maybe getting rid of this list, since targets and targets.distributions already support "many"? (or was there some other nested list that's already absent from this proposal...)

Reply:
I like this proposal, it is simpler to grok and has a nice parallel to the schema we use in the platform spec in analyzed.toml:
```
  [target]
  id = "<target identifer>"
  os = "<OS name>"
  arch = "<architecture>"
  variant = "<architecture variant>"
  [target.distribution]
  name = "<OS distribution name>"
  version = "<OS distribution version>"
```

So instead of:

```
[[targets]]
os = "<OS name>"
arch = "<architecture>"
variant = "<architecture variant>"
[[targets.distributions]]
name = "<OS distribution name>"
versions = ["version1", "version2"]
```

We would have:

```
[[targets]]
os = "<OS name>"
arch = "<architecture>"
variant = "<architecture variant>"
[[targets.distributions]]
name = "<OS distribution name>"
version = "version1"
[[targets.distributions]]
name = "<OS distribution name>"
version = "version2"
```

Reply:
Updated in 8652ec5, but I'll leave this conversation open in case others want to push back here